### DISCUSSION TITLE
Error when i run my app

### DISCUSSION THREAD

**Discussion by Melisayg12** [Homebrew/discussions#3586] [Everyday usage]
Hi, I have this error when I try to run my app. It lets me use it with version 14 of node but not with a higher one.
I put two commands:
yarn run mongo (successfully executed)
yarn run maps (maps is my app)

<img width="1296" alt="Captura de Pantalla 2022-08-30 a las 13 37 23" src="https://user-images.githubusercontent.com/99819169/187438893-b1ce1fe8-b461-4e57-89ff-832d33b330a9.png">

```

myanes@MacBook-Pro-de-Melisa-2 ipsilum % yarn run maps
A
yarn run v1.22.19
$ cd maps && env MONGO_URL=mongodb://Localhost: 27017/ipsilum NODE_TLS_REJECT_UNAUTHORIZED=0 meteor --settings .deploy/settings-dev.json --port 3004
This project uses Meteor 1.8.3, which isn't available on this platform. To work with this app on all supported platforms, use meteor update -release METEOR@2.7.3 to pin this app
to the newest compatible release.
error Command failed with exit code 1
info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.
• myanes@MacBook-Pro-de-Melisa-2 ipsilum % meteor --version
Meteor 2.7.3
© myanes@MacBook-Pro-de-Melisa-2 ipsilum%
```

**Comment by SMillerDev** [comment]
Edited the text into the question because screenshots are awful to debug.

I'm also not sure how homebrew factors into this?
